modules/order/src/Tests/OrderEmptyStoreTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\commerce_order\Tests;
+
+use Drupal\commerce\Tests\CommerceTestBase;
+use Drupal\Core\Url;
+
+/**
+ * Empty Store order page test.
+ *
+ * @group commerce
+ */
+class OrderEmptyStoreTest extends CommerceTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'commerce_product',
```

Comment:
Do we need commerce_product? Nitpick to try and keep tests a *little* faster by enabling less modules.